processor/parse/magic-block-parser.js

```diff
@@ -247,7 +248,7 @@ function tokenize(eat, value) {
             data: {
               hName: 'html-block',
               hProperties: {
-                html: json.html,
+                html: parser.transforms.html(json.html),
```

Comment:
Do we need to update non-magic block Markdown parsing too to exclude these unsafe attributes?

Reply:
The sanitizing happens for all the transformers, it's just not smart enough to know about magic blocks: 
https://github.com/readmeio/markdown/blob/next/index.js#L124

Reply:
mind adding a test for regular markdown for these cases just to be safe?